services/device-registry-mongodb/src/main/java/org/eclipse/hono/deviceregistry/mongodb/config/HttpServiceConfigProperties.java

```diff
@@ -0,0 +1,87 @@
+/*******************************************************************************
+ * Copyright (c) 2020 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+
+package org.eclipse.hono.deviceregistry.mongodb.config;
```

Comment:
this class doesn't seem to be specific to the MongoDB based registry, is it? Why not move it to the `org.eclipse.hono.service.http` package in the *service-base* module?

Reply:
It has been moved to the `org.eclipse.hono.service.http` package.